ethstore/src/dir/mod.rs

```diff
@@ -62,6 +63,8 @@ pub trait KeyDirectory: Send + Sync {
 	fn path(&self) -> Option<&PathBuf> { None }
 	/// Return vault provider, if available
 	fn as_vault_provider(&self) -> Option<&VaultKeyDirectoryProvider> { None }
+	/// Returns hash of the directory content, if supported
+	fn hash(&self) -> Result<H256, Error> { Ok(H256::zero()) }
```

Comment:
key directory does not operate the concept of "reloading accounts" at all
it just returns hash of all account headers, if it able to